lib/hooks/gitlab/templates/pipeline.html.haml

```diff
@@ -0,0 +1,37 @@
+:ruby
+  label_classes = {
+    'failed'  => 'label-danger',
+    'created' => 'label-info',
+    'manual'  => 'label-info',
+    'running' => 'label-info',
+    'success' => 'label-success',
+    'success_with_warnings' => 'label-warning'
+  }
+  web_url = payload.project.web_url
+  p_status = payload.object_attributes.status
+  p_label_class = label_classes[p_status].presence || 'label-default'
+  p_id = payload.object_attributes.id
+  p_url = "#{web_url}/pipelines/#{p_id}"
+
+%div
+  %h4
+    %span.label{class: p_label_class}
+      #{p_status}
```

Comment:
I'm not 100% sure why the right side of the badge has such an extra margin (or whitespace?) like this:

![selection_302](https://user-images.githubusercontent.com/43346/53468672-86a95980-3a9e-11e9-9b68-9f263f6a745f.png)

But perhaps this might help:

```haml
%span.label{class: p_label_class}= p_status
```

Reply:
OK.
I adjusted layout.
Thank you!